prov/efa/src/rxr/rxr_read.c

```diff
@@ -380,6 +380,7 @@ int rxr_read_post_local_read_or_queue(struct rxr_ep *ep,
 		FI_WARN(&rxr_prov, FI_LOG_EP_CTRL, "RDMA entries exhausted\n");
 		return -FI_ENOBUFS;
 	}
+	memset(read_entry, 0, sizeof(struct rxr_read_entry));
```

Comment:
This is a fairly big structure (320 bytes on x86_64 linux), we fill in most of the fields below, and the remaining fields are not necessary to fill in.  I'm not sure this is the right solution.  If you want to (during memory poisoning) memset to some weird large number to catch these issues, I'm ok with that.  But this does not feel like the right solution for critical path code.

Reply:
Fixed by only memsetting mr field, which was the problem causing field.